src/main/java/peppa/Task.java

```diff
@@ -0,0 +1,34 @@
+package peppa;
+
+public abstract class Task {
+    protected String name;
+    protected boolean done;

```

Comment:
```suggestion
    protected boolean isDone;
```
(Acc. to boolean naming conventions)